client/src/components/commonComponents/TabsCheckout/TabsCheckout.jsx

```diff
@@ -38,6 +39,7 @@ const renderStepContent = (step) => {
 const TabsCheckout = () => {
   const classes = useStyles();
   const [activeStep, setActiveStep] = useState(0);
+  const [covidAnswer, setCovidAnswer] = useState('no');
```

Comment:
Not super important but consider using constants for these string values - it's a good practice because it minimizes the chance that someone will make a mistake in the future (eg using 'No' instead of 'no' and then a comparison somewhere fails)